web/components/feed/feed-timeline-items.tsx

```diff
@@ -69,42 +65,45 @@ export const FeedTimelineItems = (props: {
   )
 
   return (
-    <Col>
+    <Col className="gap-3">
       {feedTimelineItems.map((item) => {
         // Boosted contract
-        if ('ad_id' in item) {
+        if (item.contract && ('ad_id' in item || 'contract' in item)) {
```

Comment:
Lol this is how I originally wrote it but I recently changed it bc I think it's slightly confusing to parse both ads and normal contracts in one place. I preferred to separate them as though this does introduce some duplication, it results in cleaner/easier to parse code IMO. It's fine as is, I don't feel strongly about it.

Reply:
Yeah it was annoying to keep having to duplicate changes so I just consolidated them